src/client/js/partials/breaches.js

```diff
@@ -68,6 +70,13 @@ function handleEvent (e) {
       state.emailCount = e.detail.newEmailCount
       renderZeroState()
       break
+    case e.type === 'toggle':
+      window.gtag('event', 'toggled_breach_card', {
+        action: e.target.open ? 'opened' : 'closed',
+        page_location: location.href,
```

Comment:
Q: _If_ the `location.href` includes query string data, is there any risk that this would be logging any unwanted PII that we may want to sanitize?

Reply:
This is a good point, it came up in the context of the free scan. We determined at the time that this was the only page that did this and avoided it, but I'll confirm these details. Thanks!

Reply:
Yeah, not sure if passing it through `new URL(location.href).pathname` sanitizes it enough/too much, but 🤷 &hellip;

```js
location.href
// 'https://github.com/mozilla/blurts-server/pull/3076#discussion_r1205879038?ok=computer'

new URL(location.href).pathname
// '/mozilla/blurts-server/pull/3076'
```

Reply:
Hm, so I think we do want the hostname here (traffic comes in to the same property from multiple sites), I might go with this instead: `(window.location.origin + window.location.pathname)`